### ISSUE TITLE
Message key is lost by CLI when file has only one key

### ISSUE THREAD

**Issue by eemeli** [messageformat/messageformat#339]
The current behavior **breaks the contract** provided by structured KV pairs when compiling a module, at least from the CLI. I want the precise **opposite** of the OP. 

Consider translations with just _one key_:

```yaml
# en.yaml
greet: hello
```

```yaml
# es.yaml
greet: hola
```

`pnpx messageformat <dir>`

yields =>

```js
{
  en: () => 'hello',
  es: () => 'hola'
}

```

As shown above, `greet` is **missing**. By just adding another key,

```yaml
# en.yaml
greet: hello
other: abc
```

```yaml
# es.yaml
greet: hola
other: efg
```

The correct shape is omitted:

```js
{
  en: { greet: () => 'hello', other: () => 'abc' },
  es: { greet: () => 'hola', other: () => 'efg' },
}
```

This broke me today unexpectedly where I had one translation with a single key. To keep the same structure, I now must artificially add an additional key so that the compiler does not drop valuable key/value pair information.

Is it agreed that this is/is not a problem? If this lossy behavior is _desired_, can it be opt-in only?

_Originally posted by @cdaringe in https://github.com/messageformat/messageformat/issues/328#issuecomment-893821874_

**Comment by eemeli** [comment]
@cdaringe Apologies for being so late getting to this, as it's actually an entirely separate issue from the one in the parent thread. There's an oblique reference to this "feature" in the [CLI docs](http://messageformat.github.io/messageformat/cli/#options) (in the `--simplify` option description), but I agree that it's surprising and should not be default.

**Comment by eemeli** [comment]
I'm considering an additional minor-level fix for this by expanding the `--simplify` option to provide more choices, but keeping that to not also be a breaking change is currently blocked by yargs/yargs-parser#412.

**Comment by eemeli** [comment]
Closing this, as the actual bug has long since been fixed and the upstream dependency has not shown any interest in making required changes.
